code/datums/chatmessage.dm

```diff
@@ -518,6 +528,14 @@ GLOBAL_LIST_INIT(job_colors_pastel, list(
 	else
 		message_loc = get_atom_on_turf(target)
 
+	if (LAZYLEN(message_loc.balloon_alerts))
```

Comment:
```suggestion
	if(LAZYLEN(message_loc.balloon_alerts))
```

Reply:
?
Usually there should bea space between if and the brackets following most conventions

Reply:
Ivan cried about it, I really don't care either way